core/src/main/kotlin/com/alecstrong/sql/psi/core/mysql/psi/mixins/AlterTableDropColumnMixin.kt

```diff
@@ -0,0 +1,23 @@
+package com.alecstrong.sql.psi.core.mysql.psi.mixins
+
+import com.alecstrong.sql.psi.core.mysql.psi.MySqlAlterTableDropColumn
+import com.alecstrong.sql.psi.core.psi.AlterTableApplier
+import com.alecstrong.sql.psi.core.psi.LazyQuery
+import com.alecstrong.sql.psi.core.psi.SqlCompositeElementImpl
+import com.intellij.lang.ASTNode
+
+internal abstract class AlterTableDropColumnMixin(
+    node: ASTNode
+) : SqlCompositeElementImpl(node),
+    MySqlAlterTableDropColumn,
+    AlterTableApplier {
+  override fun applyTo(lazyQuery: LazyQuery): LazyQuery {
+    return LazyQuery(
+        tableName = lazyQuery.tableName,
+        query = {
+          val columns = lazyQuery.query.columns.filter { it.element.text != node.lastChildNode.text }
```

Comment:
`node.lastChildNode` feels weird here, is there a better way?

Reply:
add a `columnName` property similar to whats in the `MySqlAlterTableChangeColumnMixin` and use that.text